README.md

```diff
@@ -35,7 +35,7 @@ Node 4.4.x and NPM 3.9.x (and above) are **required**. We suggest using [nvm](ht
   ```sh
   npm run scaffold
   ```
-  This will create `webpack/proxy.dev.js` and `src/js/config/Config.dev.js`. These files aren't tracked by git and provide a place to override proxy and general dcos-ui configuration. Edit `webpack/proxy.dev.js` to point to the  address of the `dcos-vagrant` machine.
+  This will create `webpack.proxy.dev.js` and `src/js/config/Config.dev.js`. These files aren't tracked by git and provide a place to override proxy and general dcos-ui configuration. Edit `webpack.proxy.dev.js` to point to the  address of the `dcos-vagrant` machine.
```

Comment:
I noticed that you removed the folder `webpack`, not sure what others think but I like that organization where we have something like `scripts`, `webpack` and etc.

Reply:
Having the config in a sub-folder lead to a problem with the paths, by moving it root level we don't have that problem. We can move it back into a folder later, but first I did not want to deal with that complexity

Reply:
Usually path problems can be solved by `path.join(__dirname, 'path/to/somewhere')`. Would nice to spend more time in fixing the problem than removing the organization.

Reply:
I agree with Michael, this means that we haven't set our paths correctly to begin with and is a code smell.

Reply:
@DanielMSchmidt I'm working on grouping the config again under the webpack folder. Will commit here once done

Reply:
@bstavroulakis Thanks for taking over 👍 
@weblancaster It wasn't that easy at least from my tries, having the plugins makes this a lot harder actually.